### PR TITLE
Added try/catch to return

### LINKED ISSUE #147: Unknown CLI arg fails with unhandled promise rejection
## Description
- Unknown CLI arg fails with unhandled promise rejection
- Unrelated: Also fix the usage output, it says the auth type defaults to CLI, but it should be MSI

## Expected Behavior
- We should handle this case in a more friendly way with a short output message.

## Reproduce
1. Try running helium typescript from the CLI with an unknown parameter. Like: npm start -- -fake
2. See screenshot below for output example.

## Additional Context
- Example Output: 
image

### PR DESCRIPTION
**Pull request by tobiaswright**
## Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Gracefully handle error on the CLI

## Issues Closed or Referenced

- Closes #147  (this will automatically close the issue when the PR closes)
- References #<issue number> (this references the issue but does not close with PR)

### CODE REVIEW

File: src/config/ConsoleController.ts [hunk 1]
```diff
@@ -56,8 +57,13 @@ export class ConsoleController {
         }
 
         // command line arguments
-        const args = commandLineArgs(options);
-
+        try {
+            args = commandLineArgs(options);
+        } catch(e) {
+            this.showHelp("Error: " + e.name);
+            process.exit();
```
Comment (aflinchb) [inline src/config/ConsoleController.ts:64]: Let's use the string interpolation with: `Error: ${e.name}`  to keep consistent with the rest of the file. (and I think it's cool :) ).

File: src/config/ConsoleController.ts [hunk 2]
```diff
@@ -56,8 +57,13 @@ export class ConsoleController {
...
+        }
```
Comment (aflinchb) [inline src/config/ConsoleController.ts:65]: I think we should use process.exit(-1) when it's exiting for an error.  Same with the process.exit on line 26, it should have a -1 exit code (that one was missed before).

I just checked and the default exit code for process.exit is 0 (success) so those should be the only 2 error cases here we are missing.

Comment (tobiaswright) [reply src/config/ConsoleController.ts:65]: done!

File: src/config/ConsoleController.ts (hunk 1 above)
Comment (tobiaswright) [reply src/config/ConsoleController.ts]: done!

**Pull request merged**